go.mod

```diff
@@ -128,9 +127,5 @@ replace (
 	go.opencensus.io => github.com/tilt-dev/opencensus-go v0.22.5-0.20200904175236-275b1754f353
 	golang.org/x/crypto => golang.org/x/crypto v0.0.0-20191206172530-e9b2fee46413
 
-	// Workaround for:
-	// https://github.com/moby/term/issues/15
-	golang.org/x/sys => golang.org/x/sys v0.0.0-20200826173525-f9321e4c35a6
```

Comment:
One of the updated dependencies was using a new function that wasn't available. The Darwin compilation issue in `moby/term` was fixed and the new version compiles on macOS again. I didn't see any odddities in terminal rendering using the TUI either.